packages/react-components/react-components/src/AccessibilityScenarios/Textarea.stories.tsx

```diff
@@ -0,0 +1,350 @@
+import * as React from 'react';
+
+import { Button, Textarea, Checkbox, CheckboxOnChangeData, Label } from '@fluentui/react-components';
+
+import { Scenario } from './utils';
+
+import { useForm, Controller, OnSubmit } from 'react-hook-form';
+import { usePubSub, PubSubProvider, Handler } from '@cactuslab/usepubsub';
+
+const generateCustomerId = (): string => {
+  let char;
+  let hash = 0;
+  const random = Math.random().toString();
+  for (let i = 0; i < random.length; i++) {
+    char = random.charCodeAt(i);
+    //eslint-disable-next-line no-bitwise
+    hash = (hash << 5) - hash + char;
+    //eslint-disable-next-line no-bitwise
+    hash |= 0; // Convert to 32bit integer
+  }
+  hash += 2147483647; // Convert to positive integer
+  return hash.toString().padStart(10, '0').substring(2);
+};
+
+interface FormTextareas {
+  knowledge: string;
+  effort: string;
+  problemNotSolved: string;
+  otherComments: string;
+  satisfaction: string;
+}
+
+interface FormValidation {
+  subscribe: (channel: string, handler: Handler) => () => void;
+  unsubscribe: (channel: string, handler: Handler) => void;
+}
+
+interface ValidationMessageProps {
+  id: string;
+  formValidation: FormValidation;
+}
+const ValidationMessage: React.FC<ValidationMessageProps> = ({ id, formValidation, children }) => {
+  const [isAlerting, setIsAlerting] = React.useState(true);
+
+  const alert = React.useCallback(() => {
+    setIsAlerting(false);
+    setTimeout(() => setIsAlerting(true), 200);
```

Comment:
This timeout and focus behavior causes focus to get sent back to the first field in the form that has an error, even if someone is working on an entirely different field and tabs away. In more detail: if I leave the first field in an error state to come back to later and move on, tabbing away from e.g. the 3rd field and triggering validation there will randomly send focus back to the first field.

I believe a better approach would be to only shift focus when a user activates the submit button.

Reply:
The focus should indeed move only when activating the "Send" button. When I tab through edit fields, the focus does not move to the first error field in my case. Could you please provide exact repro steps?

Reply:
First submit the form once without making any changes. Next navigate to the second textarea, and add a few characters (so the error message changes), and tab away. Your focus is then programmatically sent to the first textarea after a half-second delay.

The bug looks like it's in the `useEffect` on line 119, since after the form is submitted once, it runs again every time `errors` changes.

This same bug exists on the Input story.

Reply:
Thanks for the cue. Looks like I have fixed the bug, but I had to reintroduce the half second timeout for focusing the first error field upon form submission. Without the timeout, the focus sometimes stood at the submit button after submission.